Source/WebCore/Modules/webauthn/AuthenticatorAttestationResponse.cpp

```diff
@@ -36,6 +36,8 @@
 
 namespace WebCore {
 
+static constexpr auto useCryptoKit = UseCryptoKit::Yes;
+
```

Comment:
Why make this change?

I think you should really remove the `useCryptoKit` parameter altogether so that behavior cannot be altered at runtime, like I said before. I'm okay with you doing that as a follow up, but you shouldn't needlessly pull this out into a static variable.

Reply:
I am keeping this as this was the hook i used to develop this and it's also my worst case fallback switch in case we find an issue before shipping.

Reply:
Can you just reverting these changes so the patch is simpler?